LambdatestEcom/Pages/ProductPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DecoratorDesignPatternTests.Models;
+using LambdatestEcom.Pages;
+using Microsoft.Playwright;
+
+namespace LambdatestEcom.Pages
+{
+    internal class ProductPage
+    {
+        private readonly IPage _page;
+
+        public ProductPage(IPage page)
+        {
+            _page = page;
+        }
+
+        public async Task IncreaseQuantityOfProduct(int quantity)
```

Comment:
quantity parameter is never used. Regardless what number is provided, it is always +2